### PR TITLE
Finalize mpe search engine

### PR DESCRIPTION
**Pull request by Devessier**
Closes https://github.com/AdonisEnProvence/MusicRoom/issues/331
Closes https://github.com/AdonisEnProvence/MusicRoom/issues/332

### CODE REVIEW

File: packages/client/screens/MusicPlaylistEditorRoomsSearchScreen/index.tsx [hunk 1]
```diff
@@ -80,7 +89,7 @@ const MusicPlaylistEditorRoomsSearchScreen: React.FC<MpeTabMpeRoomsScreenProps>
             >
                 <FlatList
                     testID="library-mpe-room-search-flat-list"
-                    data={libraryRoomState.context.rooms}
+                    data={mpeRooms}
                     renderItem={({ item }) => {
                         return (
```
Comment (prastoin) [inline packages/client/screens/MusicPlaylistEditorRoomsSearchScreen/index.tsx:94]: Should we implem the playlist icons for the invited private public etc as for the mtv search cards in this pull request ?

**Review by prastoin — DISMISSED** [review]
Great job !
One question is pending but overall I'm ok with the PR
Could be ready to be merged !

**Comment by prastoin** [comment]
The `mpeRoomUniversalSearchMachine.ts` is still struggling with fetching on the first render the related data :thinking: 
I was thinking that implementing the pagination will solve the problem but I guess we will have to search deeper

File: packages/client/screens/MusicPlaylistEditorRoomsSearchScreen/index.tsx (hunk 1 above)
Comment (Devessier) [reply packages/client/screens/MusicPlaylistEditorRoomsSearchScreen/index.tsx:145]: Done in the last commit. 👍

**Review by prastoin — APPROVED** [review]
Great everything is perfect to me :smile: 
Really cool work
Ready to be merged after ci

**Pull request merged**